config/initializers/decidim.rb

```diff
@@ -1,13 +1,11 @@
 # frozen_string_literal: true
 
 Decidim.configure do |config|
-  config.expire_session_after = ENV.fetch("DECIDIM_SESSION_TIMEOUT", 90).to_i.minutes
```

Comment:
Why did you delete this config?

Reply:
Hi, this is a duplicated line, there is the same below

Reply:
But the default is now 180minutes, is it good ?

Reply:
Ok thank you for your answer! Yes 180 min is perfect!